bottender.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
Not sure if this is the right place, but I'd specify also the port in order to avoid conflicts with other applications running on the same ports.

Reply:
Port can be change on env file. 👍